### DISCUSSION TITLE
Gesture Location Relative To Parent?

### DISCUSSION THREAD

**Discussion by fweth** [pmndrs/use-gesture#527] [Ideas] [answered]
Hello, is there a feature to or would it be possible to get all gesture positions relative to the size of the container they are registered on (ranging from 0 to 1)?  What I do, when I need this, is to use `getBoundingClientRect()` and update it on `resize` events.  Of course it's always possible that some dynamic code is resizing the container without the `resize` event firing, but for most use cases, the approach seems fine to me.

**Comment by dbismut** [comment, marked as answer]
Thanks for the suggestion. 

I think it's a simple enough hook to build so that it doesnt need to be part of the lib. If it was, we would indeed have to cover all edge cases, transforms, dom mutations, it would add a lot to the bundle I'm afraid.

But we could add a recipe folder with some custom hooks that would be useful for many.
